cms/blocks.py

```diff
@@ -11,3 +12,35 @@ class ResourceBlock(blocks.StructBlock):
 
     heading = blocks.CharBlock(max_length=100)
     detail = blocks.RichTextBlock()
+
+
+class FacultyBlock(blocks.StructBlock):
+    """
+    Block class that defines a faculty member
+    """
+
+    name = blocks.CharBlock(max_length=100, help_text="Name of the faculty member.")
+    image = ImageChooserBlock(
+        help_text="Profile image size must be at least 300x300 pixels."
+    )
+    description = blocks.RichTextBlock(
```

Comment:
On a second look, is the `bio` or `description` needed? Looking at the design in the ticket it looks like only the `title` should be included apart from the `name` and `image`. Could you please confirm this?

Reply:
I tried to clone that from xPRO. xPRO also contains that same kind of title in the description.

Reply:
If the design does not show a UI element that would require this field, it should be removed

Reply:
It does show the title detail through that UI element identical to xPRO. It can't be removed and it's a required field.

Reply:
I think what @gsidebo is getting at and what I also agree with is that this could be `title = CharField(...)` rather than a rich text block. This isn't a bio section for the instructor, it's simply going to be their title/position which does not need open-ended rich text content.